src/components/RouteWrapper.tsx

```diff
@@ -584,6 +585,13 @@ export const ROUTES: RouteProperties[] = [
         description: "A computer program which generates rainbow tables to be used in password cracking.",
         category: "Password cracking and Authentication testing",
     },
+    {
+        name: "Rtgen",
+        path: "/tools/rtgen",
+        element: <Rtgen />,
+        description: "A tool for generating random network traffic to simulate various network conditions and loads.",
+        category: "Network Traffic Generation",
+    },
```

Comment:
Routewrapper needs to be in alphabetical order as well. Please put RTsort below RTgen.